administrator/language/de-DE/plg_fields_radio.ini

```diff
@@ -6,6 +6,8 @@
 
 PLG_FIELDS_RADIO="Felder - Optionsfeld"
 PLG_FIELDS_RADIO_LABEL="Optionsfeld (%s)"
+PLG_FIELDS_RADIO_PARAMS_FORM_LAYOUT_BUTTONS="Buttons"
```

Comment:
@zero-24 wir verwenden sowohl 'Buttons' als auch 'Schaltflächen' in unseren Übersetzungen.

Reply:
Hmm 'Schaltflächen' wäre wahrscheinlich das eher deutsche Wort aber "Button" scheint auch im Duden zu sein daher würde ich sagen können wir uns in Richtung Button bewegen:
https://www.duden.de/rechtschreibung/Button
https://www.duden.de/rechtschreibung/Schaltflaeche